Neos.Flow/Classes/Security/AccountIdentifier.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Flow\Security;
+
+final class AccountIdentifier
+{
+
+    /**
+     * @var string
+     */
+    protected $identifier;
+
+    /**
+     * @param string $identifier
+     */
+    public function __construct(string $identifier)
+    {
+        $this->identifier = $identifier;
+    }
```

Comment:
I would suggest to keep the constructor private and add a
```
public static fromString(string $identifier): self
{
    return new static($identifier);
}
```
That will allow us to add more constructors lateron

Reply:
But only for this class? And not for the others (AuthenticationPRoviderName, CredentialsSource)?

Reply:
Yes, your're right. lets do it for all new VOs

Reply:
Fixed in 85614b7